packages/app/src/internal/registry/namespace.ts

```diff
@@ -0,0 +1,292 @@
+/*
```

Comment:
why are there diffs to registry/namespace.js and registry/namespace.ts in this set of diffs?

Related:  was expecting to see deletes of each associated .js file, then I think github will recognize them largely as file moves and show things as a diff between the two files vs whole-new-file-green / while-new-file-red for easier review

Reply:
We still have some  build errors at the moment so using for reference. 

I assume we are just going to delete the JS folders and repoint the project to the new `dist` folder?

Reply:
Yeah, all the libs that use typescript can be used as a pattern, e.g. react-native-device-info, microsoft/reactxp etc - they have `src/` or `lib/` with all the `.ts` files, and then in package.json they point at `dist/` and in the community packages at least bob the builder :construction_worker:  is used in a package.json script to do the transpile

building from source in place then (like a `patch-package` case where a user needs the source to patch) involves a postinstall step *after* patch-package where they will have to run the build script locally after patch-package runs  since we shouldn't commit dist/ we just ship it in releases